appspec.yml

```
@@ -1,4 +1,98 @@
-version: 0.0
+                - name: Setup Go environment
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 16

(indentation)